appyters/lncHUB2/lncHUB2.ipynb

```diff
@@ -506,7 +559,7 @@
    "outputs": [],
    "source": [
     "%%appyter markdown\n",
-    "Below we list the top 20 lncRNAs, out of all 15,862 lncRNAs within our database, that correlate most with {{query.raw_value}} based on their Pearson correlation coefficients."
+    "Below we list the top 20 lncRNAs, out of all {{num_lnc}} lncRNAs within our database, that correlate most with {{query.value[0].raw_value}} based on their Pearson correlation coefficients."
```

Comment:
If you load the num_lnc at runtime, this will need to be `display(Markdown(`-ed at runtime as well